packages/vuetify/src/components/VSelect/VSelect.js

```diff
@@ -387,6 +389,7 @@ export default VTextField.extend({
       input.data.domProps.value = null
       input.data.attrs.readonly = true
       input.data.attrs['aria-readonly'] = String(this.readonly)
+      input.data.on.keypress = this.onKeyPress
```

Comment:
keypress is deprecated, move to the keydown handler

Reply:
ok, i will change it soon

Reply:
@KaelWD so I tried to use keydown handler, but I couldn't find good way to detect if pressed key is printable (https://stackoverflow.com/q/43877434/). Looks like proper solution would be to use [`beforeinput`](https://developer.mozilla.org/en-US/docs/Web/Events/beforeinput) event, but it has poor browser support ([not implemented in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=970802)). Maybe it is good idea to keep `keypress`...

Reply:
That's a bit silly, guess it'll have to do then.